src/Illuminate/Console/Scheduling/ScheduleRunCommand.php

```diff
@@ -127,13 +129,17 @@ protected function runEvent($event)
 
         $start = microtime(true);
 
-        $event->run($this->laravel);
+        try {
+            $event->run($this->laravel);
 
-        $this->dispatcher->dispatch(new ScheduledTaskFinished(
-            $event,
-            round(microtime(true) - $start, 2)
-        ));
+            $this->dispatcher->dispatch(new ScheduledTaskFinished(
+                $event,
+                round(microtime(true) - $start, 2)
+            ));
 
-        $this->eventsRan = true;
+            $this->eventsRan = true;
+        } catch (Throwable $e) {
+            app(ExceptionHandler::class)->report($e);
```

Comment:
This should be typehinted in the `handle` method just like the schedule and the dispatcher.

Reply:
I agree. @michaeldyrynda can you adjust that?

Reply:
Good call, done!